utils/navigator_msgs/msg/Hosts.msg

```diff
@@ -0,0 +1 @@
+string[] hosts
```

Comment:
You should take more  advantage of ros msgs here. Ros msgs can be maps (dictionaries), so it isn't so unclear what the format of the data in this message is.